### PR TITLE
Bring phoenix benchmark code inline with current versions

### PR DESCRIPTION
**Pull request by djinn**
also production build with distillery

### CODE REVIEW

File: toolset/setup/linux/languages/erlang.sh [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+VERSION="20.1-1"
 RELEASE="trusty"
 ARCH="amd64"
 
-fw_get -O http://packages.erlang-solutions.com/site/esl/esl-erlang/FLAVOUR_2_esl/esl-erlang_${VERSION}~ubuntu~${RELEASE}_${ARCH}.deb
+fw_get -O http://packages.erlang-solutions.com/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_${VERSION}~ubuntu~${RELEASE}_${ARCH}.deb
 dpkg -x esl-erlang_${VERSION}~ubuntu~${RELEASE}_${ARCH}.deb $IROOT/erlang
 $IROOT/erlang/usr/lib/erlang/Install -minimal $IROOT/erlang/usr/lib/erlang
-
+fw_get -O https://s3.amazonaws.com/rebar3/rebar3 && mv rebar3 $IROOT/erlang/usr/bin && chmod +x $IROOT/erlang/usr/bin/rebar3
```
Comment (msmith-techempower) [inline toolset/setup/linux/languages/erlang.sh]: Is there any problem with changing the name to `rebar` and replacing the packaged version?

`rebar3` being called everywhere feels noisy.

Comment (djinn) [reply toolset/setup/linux/languages/erlang.sh]: The rebar does not come packaged with 20.1 release of erlang.

**Comment by nbrady-techempower** [comment]
@djinn looks like all erlang tests are breaking. would you mind taking a look? im out until wednesday and then can help out if needed

**Comment by djinn** [comment]
I will fix it

**Pull request closed without merging**